gen/nodes.ts

```diff
@@ -845,6 +845,7 @@ export interface ObjectTypeAnnotation extends Omit<FlowType, "type"> {
   properties: (K.ObjectTypePropertyKind | K.ObjectTypeSpreadPropertyKind)[];
   indexers: K.ObjectTypeIndexerKind[];
   callProperties: K.ObjectTypeCallPropertyKind[];
+  inexact: boolean | undefined;
```

Comment:
Now that we're generating TypeScript interfaces for all the AST definitions, you can see the consequences of changes immediately!